plotnine/tests/test_annotate.py

```diff
@@ -19,3 +21,30 @@ def test_multiple_annotation_geoms():
                   yend=4.2, color='red', size=1))
 
     assert p == 'multiple_annotation_geoms'
+
+def test_multiple_annotation_geoms_using_classes():
+    from plotnine import geom_point, geom_text, geom_rect, geom_segment
+    p = (ggplot(df, aes('x', 'y')) +
+         geom_point() +
+         annotate(geom_point, 0, 1, color='red', size=5) +
+         annotate(geom_text, 1, 2, label='Text', color='red',
+                  size=15, angle=45) +
+         annotate(geom_rect, xmin=1.8, xmax=2.2, ymin=2.8,
+                  ymax=3.2, size=1, color='red', alpha=0.3) +
+         annotate(geom_segment, x=2.8, y=3.8, xend=3.2,
+                  yend=4.2, color='red', size=1))
+
+    assert p == 'multiple_annotation_geoms'
```

Comment:
Maybe alter the previous test, so that two use strings, 2 use instances of geom.